R/ui.R

```diff
@@ -142,7 +142,10 @@ style_file <- function(path,
                         ...,
                         style = tidyverse_style,
                         transformers = style(...)) {
-  map_lgl(path, style_file_one, ..., style = style, transformers = transformers)
+  changed <- map_lgl(path,
+    style_file_one, ..., style = style, transformers = transformers
+  )
+  invisible(tibble(file = path, changed = changed))
```

Comment:
Is this return value used anywhere?

Reply:
No, I just found `style_file_one()` is redundant anyways with the current implementation of `transform_files()`. Must have overlooked that at some point. I will make the changes in `transform_files()`